faststream/rabbit/broker/broker.py

```diff
@@ -655,3 +655,13 @@ async def declare_exchange(
         """Declares exchange object in **RabbitMQ**."""
         assert self.declarer, NOT_CONNECTED_YET  # nosec B101
         return await self.declarer.declare_exchange(exchange)
+    
+    async def ping(self, timeout: int | None) -> bool:
+        await super().ping(timeout)
+        try:
+            channel = await self._connection.channel()
+            await channel.declare_queue(name='test_queue', durable=True)
```

Comment:
We should no create side-effects inside broker. Please, use native `ping` method (if it exists) or check underlying library flag

Reply:
In RMQ not ping method, add check flags